src/components/congratulations.vue

```diff
@@ -0,0 +1,45 @@
+<template>
```

Comment:
ah, on my computer I had made this change, but forgot to make it consistent with github